### ISSUE TITLE
Recent Update of login API

### ISSUE THREAD

**Issue by Marindrew** [fastlane/itc-api-docs#13]
So, Apple updates their APIs

Now the Login URL is
https://idmsa.apple.com/appleauth/auth/signin
with appliation/json POST request with the following parameters:
{"accountName":"email@me.com","password":"password","rememberMe":true}

it will respond with "X-SESS", "site" and "aa" cookie. After this point I am stuck, as the website somehow gets "wosid" cookie, couldn't track it so far.

Any suggestions?

**Comment by KrauseFx** [comment]
Yes, it's already been implemented on spaceship: https://github.com/fastlane/spaceship/commit/d55a43ae74777d52ec45d39933ec07dd89d6f1a7

**Comment by TiernanKennedy** [comment]
@Marindrew did you get any further with this? I looked at the code on spaceship but it wasn't obvious how to get it working. I am getting back cookies from that URL but not sure what's need to auth correctly.

We run our scripts with Python, hence why Spaceship isn't ideal for the use case.

**Comment by diaojunxian** [comment]
@TiernanKennedy Hi TiernanKennedy, can you help me ,I want to know how to use the api with Python, as konw little about Ruby ,I felt very confuse.I hope you can give me a simple example to use the api, Thanks very very much.
